src/component/time-active/TimeActive.jsx

```diff
@@ -0,0 +1,173 @@
+import React, {Fragment} from 'react';
+import PropTypes from "prop-types";
+import {Card} from "@material-ui/core";
+import ActiveTimeStyles from "./time-active-styles";
+import Typography from '@material-ui/core/Typography';
+import withStyles from "@material-ui/core/es/styles/withStyles";
+import TextField from "@material-ui/core/es/TextField/TextField";
+import {TemplateConst} from "../time-template/template-const";
+import PageSplit from "../page-split/PageSplit";
+import {PlayArrow, Stop} from "@material-ui/icons";
+import Fab from "@material-ui/core/Fab";
+import TemplateListConst from "../template-list/template-list-const";
+
+class TimeActive extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  addLeadingZero(number) {
+    return (number < 10) ? '0' + number : number;
+  };
+
+  getSkillsList(skills = []) {
+    let skillsString = '';
+    for (let i = 0; i < skills.length; i++) {
+      if (skillsString.length === 0) {
+        skillsString += skills[i].name;
+      } else {
+        skillsString += `, ${skills[i].name}`;
+      }
+    }
+    return skillsString;
+  };
+
+  formatDuration(duration) {
+
+    const second = this.addLeadingZero(Math.floor((duration / 1000) % 60));
+    const minute = this.addLeadingZero(Math.floor((duration / (1000 * 60)) % 60));
+    const hour = this.addLeadingZero(Math.floor((duration / (1000 * 60 * 60))));
+
+    return `${hour}:${minute}:${second}`;
+  };
+
+  updateTime() {
+    const duration = this.props.activeTimeEvent.duration;
+    if (typeof duration !== "undefined") {
+      this.props.updateActiveTimeEventDuration(duration + 1000);
+    }
+  }
+
+  shouldComponentUpdate(nextProps, nextState, nextContext) {
+    if (this.props.activeTimeEvent.id !== nextProps.activeTimeEvent.id) {
+      clearInterval(this.interval);
+      if (nextProps.activeTimeEvent.stop_time == null) {
+        this.interval = setInterval(() => this.updateTime(), 1000);
+      }
+    } else {
+      if (nextProps.activeTimeEvent.stop_time != null) {
+        clearInterval(this.interval);
+      }
+    }
+    return true;
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.interval);
+  }
+
+  render() {
+
+    const {classes, handleStartStopClick} = this.props;
+    const currentDay = new Date().getDate();
+    const currentMonth = (new Date().getMonth()) + 1;
+
+    const {
+      skills,
+      templateName,
+      organization_name: organizationName,
+      project_name: projectName,
+      activity,
+      template_id: templateId,
+      stop_time: stopTime,
+    } = this.props.activeTimeEvent;
+
+    console.log('activeTimeEvent',this.props.activeTimeEvent)
```

Comment:
we don't want to log anything. Please remove the console.log

Reply:
Done